dvc/schema.py

```diff
@@ -16,10 +17,11 @@
     StageParams.PARAM_ALWAYS_CHANGED: bool,
 }
 
+DATA_SCHEMA = {**CHECKSUMS_SCHEMA, Required("path"): str}
 LOCK_FILE_STAGE_SCHEMA = {
     Required(StageParams.PARAM_CMD): str,
-    Required(StageParams.PARAM_DEPS): {str: output.CHECKSUM_SCHEMA},
-    Required(StageParams.PARAM_OUTS): {str: output.CHECKSUM_SCHEMA},
+    StageParams.PARAM_DEPS: [DATA_SCHEMA],
```

Comment:
I suppose this doesn't support `dvc import`, right? For it it needs `repo: ...` key

Reply:
Yup, import stages are not supported.

Reply:
Did not we discuss to keep it single stage for now? And, to discuss those later on?

Reply:
@skshetry Ah, yeah, that's a good point! My bad.